config.yaml

```diff
@@ -78,3 +78,7 @@ options:
     description: Specifies the enabled cipher suites to be used in ZooKeeper TLS negotiation (csv). Overrides any explicit value set via the zookeeper.ssl.ciphersuites system property (note the single word "ciphersuites"). The default value of null means the list of enabled cipher suites is determined by the Java runtime being used.
     type: string
     default: ""
+  profile:
+     description: 'Profile representing the scope of deployment, and used to enable high-level customisation of sysconfigs, resource checks/allocation, warning levels, etc. Allowed values are: “production”, “staging” and “testing”'
+     type: string
+     default: production
```

Comment:
I'm wondering whether for the time being it is easier to set the defautl to test and tell to field to set it to production for the deployment

Reply:
I'm not sure I agree. It's not blocking, just will cause loads of ugly log messages. But we would want to 'warn' users of their setting by default, so I think it makes sense to keep it here.

Reply:
I'm just not sure about the specs of the github runners whether they would comply with this. If the tests run, more than fine to keep it as is

Reply:
Yup me neither! Will wait and see :P